lib/utils/tar.js

```diff
@@ -60,7 +60,12 @@ const getContents = async (manifest, tarball) => {
       totalEntrySize += entry.size
       const p = entry.path
       if (p.startsWith('package/node_modules/')) {
```

Comment:
```suggestion
      if (p.startsWith('package/node_modules/') && p !== 'package/node_modules/') {
```

Reply:
The regex falls over if there are NO characters after `package/node_modules/`

Reply:
Suggestion applied 👍🏽